### DISCUSSION TITLE
DEV Branch

### DISCUSSION THREAD

**Discussion by N3M1X10** [N3M1X10/zapret_build#39] [General]
### [DEV Branch](https://github.com/N3M1X10/zapret_build/tree/dev) 

Изменения в dev: https://github.com/N3M1X10/zapret_build/compare/main...dev

Обсуждение меж-релизных изменений

**Comment by N3M1X10** [comment]
https://github.com/N3M1X10/zapret_build/compare/main...dev

в текущей службе general отвалилось медиа в дискорде и не грузят видео в ютубе

- Не могу отправлять медиа с extended
- Простой текст в дискорде грузит
- Голос в дс работает
- В Ютуб не грузит видео

**Reply by Eptiy** [reply]
> * Не могу отправлять медиа с extended

> * В Ютуб не грузит видео

Добавление в этот лист доменов из твоих листов для дс и ют фиксят проблемы у меня:
[list-extended.txt](https://github.com/user-attachments/files/17381479/list-extended.txt)

В гите листа тред уже создавал по ют, добавлю инфу про медиа в дс.

**Reply by N3M1X10** [reply]
добавил general в extended
и батник с extended

**Reply by N3M1X10** [reply]
Лично у меня часть дискорда отваливается и не грузит с этим батником, list general работает

**Reply by N3M1X10** [reply]
Либо временный лаг, сейчас проверил, грузит

**Reply by Eptiy** [reply]
По сути это тот же старый extended, только меньше. Не знаю даже, может ли он ломать дс, учитывая, что домены из general добавили. Сам позже основательно проверю в дс.

**Reply by Eptiy** [reply]
Лист обновился, проверил, всё работает.

**Comment by N3M1X10** [comment]
Переделал сборку под парсер, установка служб теперь через один батник
Отдельное спасибо тому кто первым додумался стряпнуть это сюда

**Reply by N3M1X10** [reply]
Просьба протестировать работоспособность нового варианта

**Reply by Eptiy** [reply]
Работает.
